pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -98,6 +98,25 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		disableFeatureGates()
 	})
 
+	It("when spec validator pass, should allow", func() {
```

Comment:
There are still network-related tests in this file which still pass due to fact that the network validator is still used in `ValidateVirtualMachineInstanceSpec`.
Please consider passing the spec validation function when instantiating the VMICreateAdmitter.

Reply:
Since the injection include the network-validator.Validate() (this is the code `ValidateVirtualMachineInstanceSpe` use)  as well, network-related tests can be moved outside this file.

I moved relevant network-tests to pkg/network, note that probes related tests are not covered at the moment by the network-validator thus remained in this file.

Reply:
Thank you @ormergi for pointing out that the network related tests in `pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go` do not go through the admitter, but are testing an implementation detail (ValidateVirtualMachineInstanceSpec).
That makes my comment invalid, so please ignore it and resolve this thread.